pkg/apis/pipeline/pod/template.go

```diff
@@ -146,3 +146,96 @@ func (tpl *Template) ToAffinityAssistantTemplate() *AffinityAssistantTemplate {
 		ImagePullSecrets: tpl.ImagePullSecrets,
 	}
 }
+
+// PodTemplate holds pod specific configuration
+type PodTemplate = Template
```

Comment:
This is causing https://prow.tekton.dev/view/gs/tekton-prow/pr-logs/pull/tektoncd_pipeline/5329/pull-tekton-pipeline-build-tests/1559554478363906048#1:build-log.txt%3A1366 to fail - you should add `//nolint:revive` to the end of this line, since I don't see a problem with the `pod.PodTemplate` stuttering that revive is complaining about.

Reply:
Thanks Andrew!